src/ReleaseApp/Infrastructure/Shared/Mapper/ReleaseGroupDtoCollectionMapper.php

```diff
@@ -78,4 +88,37 @@ protected function getReleaseGroupLink(int $id): string
     {
         return sprintf(ReleaseAppConstant::RELEASE_GROUP_LINK_PATTERN, $this->configurationProvider->getReleaseAppUrl(), $id);
     }
+
+    /**
+     * @param \ReleaseApp\Domain\Entities\Collection\UpgradeInstructionModuleCollection $moduleCollection
+     * @param \ReleaseApp\Domain\Entities\UpgradeInstructionMeta $meta
+     *
+     * @return \ReleaseApp\Domain\Entities\Collection\UpgradeInstructionModuleCollection
+     */
+    protected function applyMeta(
+        UpgradeInstructionModuleCollection $moduleCollection,
+        UpgradeInstructionMeta $meta
+    ): UpgradeInstructionModuleCollection {
+        if (!$meta->getInclude()->isEmpty()) {
```

Comment:
use early return

if ($meta->getInclude()->isEmpty()) {
    return ....
}

Reply:
early return won't work here unless the logic will be extracted into two additional methods
`applyIncludes()` and `applyExcludes()`. Then all the logic can be moved there and early return should be used.

Reply:
Removed these condition